generator/plugins/graphql/plugin.go

```diff
@@ -35,6 +35,71 @@ func (p *Plugin) Init(config *generator.GenerateConfig, plugins []generator.Plug
 	}
 	p.schemaConfigs = cfgs
 	p.generateCfg = config
+
+	err = p.parseImports()
+	if err != nil {
+		return errors.Wrap(err, "failed to decode imports")
+	}
+
+	return nil
+}
+
+func (p *Plugin) parseImports() error {
+	for _, pluginsConfigsImports := range p.generateCfg.PluginsConfigsImports {
+		cfg := new([]*SchemaConfig)
+		err := mapstructure.Decode(pluginsConfigsImports[SchemasConfigsKey], cfg)
+
+		if err != nil {
+			return errors.Wrap(err, "failed to decode config")
+		}
+
+		for _, schema := range *cfg {
+			err := p.parseImportedSchema(schema)
+
+			if err != nil {
+				return err
+			}
+		}
+	}
+
+	return nil
+}
+
+func (p *Plugin) parseImportedSchema(cfg *SchemaConfig) error {
+	var importedSchemaName = cfg.Name
```

Comment:
`cfg` could be nil, so I would like to suggest you to check the state

Reply:
done, thanks.